### DISCUSSION TITLE
ESMv1 model coupling run stops at ocn read data step when running

### DISCUSSION THREAD

**Discussion by jinboxie** [E3SM-Project/E3SM#7770] [E3SM model help]
I'm trying to do a coupled run using the following run at the commit of d123ccfc783833e5041283b5e78fd6e2fe1b2090 (HEAD), with the following procedure on LLNL's LC machine quartz (36 pes/node). I used the configuration of following:
set my_npes        = 72
set walltime       = 02:00:00
set compset        = A_WCYCL20TRS_CMIP6 
set resolution     = ne30_oECv3_ICG  #ne30_oEC  #ne30_oECv3
set machine        = quartz
I seem to get error before running start the simulations as follows:

### **From e3sm.log**
Image              PC                Routine            Line        Source
e3sm.exe           00000000031ECFEA  Unknown               Unknown  Unknown
e3sm.exe           000000000301F7A1  pio_support_mp_pi         120  pio_support.F90
e3sm.exe           000000000301D217  pio_utils_mp_bad_          97  pio_utils.F90
e3sm.exe           0000000003002103  nf_mod_mp_pio_inq         277  nf_mod.F90
e3sm.exe           000000000179E34C  mpas_io_mp_mpas_i         315  mpas_io.f90
e3sm.exe           000000000174CE7D  mpas_bootstrappin         525  mpas_bootstrapping.f90
e3sm.exe           00000000026CF7FE  ocn_comp_mct_mp_o         517  ocn_comp_mct.f90
e3sm.exe           0000000000435F99  component_mod_mp_         269  component_mod.F90
e3sm.exe           000000000042560E  cime_comp_mod_mp_        1192  cime_comp_mod.F90
e3sm.exe           00000000004330EC  MAIN__                     92  cime_driver.F90
e3sm.exe           0000000000417072  Unknown               Unknown  Unknown
libc-2.17.so       00002AAAAD1E3555  __libc_start_main     Unknown  Unknown
e3sm.exe           0000000000416F69  Unknown               Unknown  Unknown

###  **From ocn.log**
----- done parsing run-time I/O from streams.ocean -----
 Reading dimensions from input streams ...

   *** unable to open input file mesh.nc for stream 'input_init'

 ----- skipping inactive stream 'mesh'

 ----- reading dimensions from stream 'input' using file /p/lustre2/xie7/e3sm/model/new/simulations/quartz.E3SM.NAT_forcing_test_72nodes.ne30_oECv3_ICG/run/mpaso.rst.1945-01-01_00000.nc
        nCells = 235160
        nEdges = 714274
        nVertices = 478835
        TWO = 2
        maxEdges = 7
        maxEdges2 = 14
        vertexDegree = 3
        nVertLevels = 60

 ----- reading dimensions from stream 'restart' using file mpaso.rst.1945-01-01_00000.nc
        nCells = 235160
        nEdges = 714274
        nVertices = 478835
        TWO = 2
        maxEdges = 7
        maxEdges2 = 14
        vertexDegree = 3
        nVertLevels = 60
        nVertLevelsP1 = 61
        nForcingGroupsMax = 4

Does anybody know what the problem is? # #

**Comment by jonbob** [comment]
@jinboxie - that's a pretty big run to try on 72 pes. You might be blowing memory? We tend to run it on 1000+ pes, though it really depends on platform specifics
